app/extras/group_discussions_viewer.rb

```diff
@@ -2,7 +2,7 @@ class GroupDiscussionsViewer
   def self.for(group: nil, user: nil, filter: nil)
     groups = groups_displayed(group: group, user: user)
     Queries::VisibleDiscussions.new(groups: groups, user: user).tap do |discussions|
-      discussions = discussions.unread if filter == 'unread'
+      discussions = discussions.unread if filter == 'show_unread'
```

Comment:
I waffled about this some; I think we're going to want to put more filters on down the line, and wanted a common name that would make sense for them ('show_muted', 'show_participating', 'show_authored_by_me', whatever)